freva-data-portal-worker/src/data_portal_worker/utils.py

```diff
@@ -7,7 +7,7 @@
 from socket import gethostname
 from typing import Optional
 
-import appdirs
```

Comment:
Just out of curiousity why was this necessary?

Reply:
When I was setting up development env on my machine i faced an issue regarding using this lib and when I searched a bit, I found this lib is gonna be deprecated in the future and be replaced with `platformdirs`; as an example conda already has been started replacing ...

https://github.com/conda/conda/issues/13306

Reply:
Ok, I see!